tests/lib/newlib/newlib_api/src/main.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/time.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <ztest_error_hook.h>
+#include <toolchain.h>
+#include <stdio.h>
+#include <stddef.h>
+
+extern int _gettimeofday(struct timeval *__tp, void *__tzp);
+extern int *__errno(void);
+extern void __chk_fail(void);
+extern void *_sbrk(intptr_t count);
+extern int _read(int fd, char *buf, int nbytes);
+extern int _write(int fd, const void *buf, int nbytes);
+extern int _open(const char *name, int mode);
+extern int _close(int file);
+extern int _lseek(int file, int ptr, int dir);
+extern int _isatty(int file);
+extern int _kill(int i, int j);
+extern int _getpid(void);
+extern int _fstat(int file, struct stat *st);
+extern void _exit(int status);
+extern void __stdin_hook_install(unsigned char (*hook)(void));
+extern int _stdout_hook_default(int c);
+
+#define TEST_INPUT 1
+
+/**
+ * @brief Test newlib apis without CONFIG_POSIX_API configuration
+ *
+ * @details Verify the apis which are implemented in libc/newlib/.
+ * And check it could be invoked which is default implemented with
+ * different input.
+ *
+ * @see _gettimeofday(), __errno(), _isatty(), _kill()
+ * _getpid(), _fstat()
+ */
+void test_newlib_api(void)
+{
+#ifndef CONFIG_POSIX_API
+	ztest_test_skip();
```

Comment:
> `@brief Test newlib apis without CONFIG_POSIX_API configuration`

The description does not match the implemented behaviour.

Reply:
Fixed.